falcon/request.py

```diff
@@ -1028,6 +1060,22 @@ def _parse_form_urlencoded(self):
 
             self._params.update(extra_params)
 
+    def _parse_rfc_forwarded(self):
+        """rfc 7239"""
```

Comment:
Style: 
* `rfc` ==> `RFC`
* The first line of a comment should end in a period.